wasm/src/glob-compiler.cc

```diff
@@ -128,7 +128,7 @@ std::string glob_translate(const char *glob, size_t n, const std::vector<std::st
 
     if (delimiters.empty())
     {
-        single_mark = ".";
+        single_mark = "[^\\.]";
```

Comment:
☝️ Here's where we ensure the default behaviour matches what would happen if `["."]` was passed.